packages/ibm-products/src/components/BigNumbers/constants.js

```diff
@@ -64,7 +64,13 @@ export const getIconSize = (size) => {
  *
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#locales
  */
-export const getSupportedLocale = (locale) =>
-  Intl.NumberFormat.supportedLocalesOf(locale).length > 0
-    ? locale
-    : DefaultLocale;
+export const getSupportedLocale = (locale) => {
```

Comment:
Updated to `try/catch` because
```
Intl.NumberFormat.supportedLocalesOf(locale);
```
will throw an error if an unsupported locale string is used.

Reply:
this looks like a handy utility. i would move this into the global utilities folder!

Reply:
Done.